testsuite/integration/microprofile/src/test/java/org/wildfly/test/integration/microprofile/health/MicroProfileHealthUtils.java

```diff
@@ -77,4 +80,33 @@ static void testManagementOperation(ModelNode response, boolean mustBeUP, String
             fail("Probe named " + probeName + " not found in " + result);
         }
     }
+
+    static List<JsonObject> retrieveHttpEndPointChecks(final String healthURL) throws IOException {
```

Comment:
don't duplicate code, reuse it in the original methods

Reply:
Done, thanks